code/edgeconsent/src/main/java/com/adobe/marketing/mobile/edge/consent/Consents.java

```diff
@@ -185,7 +185,7 @@ public boolean equals(final Object comparingConsentObject) {
 
 	/**
 	 * Compares the current consent instance the with the passed object ignoring the timestamp field in metadata
-	 *
+	 * @param comparingConsent the current consents instance
```

Comment:
the current consents instance is "this", while comparingConsent is the new consent object to be compared against current consent settings